R/utils.R

```diff
@@ -346,7 +346,7 @@ pm_labs <- function(...) {
 
 noline <- ggplot2::element_blank()
 
-merge.list <- function(x,y,...,open=FALSE,
+merge_list <- function(x,y,...,open=FALSE,
```

Comment:
Just updated packages and I think roxygen is now flagging `merge.list()` as an S3 method. It's super old and I don't think it was intended to be used as S3 method. I just converted it to `merge_list()` instead:

```
➜  metrumresearchgroup/pmplots/R (rotate-at)  grep merge *.R
utils.R:  x <- merge.list(x,x0)
utils.R:  x <- merge.list(x,x0)
utils.R:  x <- merge.list(x,x0)
utils.R:merge.list <- function(x,y,...,open=FALSE,
```

```
Rscript inst/script/document.R
ℹ Updating pmplots documentation
ℹ Loading pmplots
Loading required package: ggplot2
✖ utils.R:349: S3 method `merge.list` needs @export or @exportS3method tag.
make build
```